src/plugins/rpcmon/rpcmon.cpp

```diff
@@ -275,6 +275,35 @@ static std::optional<rpc_info_t> parse_MIDL_STUBLESS_PROXY_INFO(drakvuf_t drakvu
     return parse_MIDL_STUB_DESC(drakvuf, info, rpc_proxy_info_addr);
 }
 
+static std::optional<uint64_t> parse_FORMAT_STRING(drakvuf_t drakvuf, drakvuf_trap_info* info, addr_t arg)
+{
+    auto vmi = vmi_lock_guard(drakvuf);
+
+    // See https://docs.microsoft.com/en-us/windows/win32/rpc/procedure-header-descriptor
+    constexpr int oi_flags_field_offset = 0x1;
```

Comment:
I think this would be better to be moved to `private.h` and declared using `#define`

Reply:
Done